linker-interface/shared/src/main/scala/org/scalajs/linker/interface/ReportToLinkerOutputAdapter.scala

```diff
@@ -151,7 +151,10 @@ object ReportToLinkerOutputAdapter {
     }
   }
 
-  private val sourceMapRe = """(?m)^//# sourceMappingURL=.*$""".r
+  /* This regex would normally be written with the (?m) flag, but that would
+   * require ES2018, so we work around it.
+   */
+  private val sourceMapRe = """(?:^|\n)(//# sourceMappingURL=[^\n]*)(?:\n|$)""".r
```

Comment:
Won't this fail on windows style newlines?

Reply:
It would, but we never emit Windows-style newlines. We always use Unix-style ones.